### PR TITLE
Some renaming

### PR DESCRIPTION
**Pull request by kostekIV**
_A -> amplification_coefficient
BNB -> native
_LP -> liquidity_pool

### CODE REVIEW

File: eth/contracts/StableSwap/StableSwapTwoPool.sol [hunk 1]
```diff
@@ -96,19 +97,33 @@ contract StableSwapTwoPool is Ownable, ReentrancyGuard {
...
     event NewFee(uint256 fee, uint256 admin_fee);
-    event RampA(uint256 old_A, uint256 new_A, uint256 initial_time, uint256 future_time);
+    event RampA(
+        uint256 old_A,
+        uint256 new_A,
+        uint256 initial_time,
+        uint256 future_time
+    );
     event StopRampA(uint256 A, uint256 t);
-    event SetBNBGas(uint256 bnb_gas);
```
Comment (Marcin-Radecki) [inline eth/contracts/StableSwap/StableSwapTwoPool.sol:111]: Btw is this a global variable `bnb_gas` or local parameter to an event?

File: eth/contracts/StableSwap/StableSwapTwoPool.sol [hunk 2]
```diff
@@ -124,58 +139,64 @@ contract StableSwapTwoPool is Ownable, ReentrancyGuard {
     /**
      * @notice initialize
      * @param _coins: Addresses of ERC20 conracts of coins (c-tokens) involved
-     * @param _A: Amplification coefficient multiplied by n * (n - 1)
+     * @param _amplification_coefficient: Amplification coefficient multiplied by n * (n - 1)
      * @param _fee: Fee to charge for exchanges
      * @param _admin_fee: Admin fee
      * @param _owner: Owner
-     * @param _LP: LP address
```
Comment (Marcin-Radecki) [inline eth/contracts/StableSwap/StableSwapTwoPool.sol:131]: So this is a liquidity pool or liquidity provider?

File: eth/contracts/StableSwap/StableSwapTwoPool.sol [hunk 3]
```diff
@@ -124,58 +139,64 @@ contract StableSwapTwoPool is Ownable, ReentrancyGuard {
...
         address _owner,
-        address _LP
+        address _liquidity_pool
     ) external {
         require(!isInitialized, "Operations: Already initialized");
         require(msg.sender == STABLESWAP_FACTORY, "Operations: Not factory");
-        require(_A <= MAX_A, "_A exceeds maximum");
+        require(
+            _amplification_coefficient <= MAX_AMPLIFICATION_COEFFICIENT,
+            "_A exceeds maximum"
```
Comment (Marcin-Radecki) [inline eth/contracts/StableSwap/StableSwapTwoPool.sol:160]: Here we can rename `_A` as well

File: eth/contracts/StableSwap/StableSwapTwoPool.sol [hunk 4]
```diff
@@ -359,7 +401,13 @@ contract StableSwapTwoPool is Ownable, ReentrancyGuard {
...
+        emit AddLiquidity(
+            msg.sender,
+            amounts,
+            fees,
+            D1,
+            token_supply + mint_amount
+        );
     }
 
     function get_y(
```
Comment (Marcin-Radecki) [inline eth/contracts/StableSwap/StableSwapTwoPool.sol:413]: We won't touch this `i, `j` .. here? This is the function we'll call in MOST contract?

File: eth/contracts/StableSwap/StableSwapTwoPool.sol (hunk 1 above)
Comment (kostekIV) [reply eth/contracts/StableSwap/StableSwapTwoPool.sol:111]: Its like a fn signature, so just named variable

File: eth/contracts/StableSwap/StableSwapTwoPool.sol (hunk 2 above)
Comment (kostekIV) [reply eth/contracts/StableSwap/StableSwapTwoPool.sol:131]: liquidity pool

File: eth/contracts/StableSwap/StableSwapTwoPool.sol (hunk 4 above)
Comment (kostekIV) [reply eth/contracts/StableSwap/StableSwapTwoPool.sol:413]: In most we will call exchange, in the frontend we will cal one of this or get_dy.

File: eth/contracts/StableSwap/StableSwapTwoPool.sol (hunk 3 above)
Comment (kostekIV) [reply eth/contracts/StableSwap/StableSwapTwoPool.sol]: done

File: eth/contracts/StableSwap/StableSwapTwoPool.sol (hunk 4 above)
Comment (kostekIV) [reply eth/contracts/StableSwap/StableSwapTwoPool.sol:413]: changed this in get_dy and exchange function. Now its mixed up trhough the file but consistent on the subset we will use. Not sure if we want to change everything?

Comment (Marcin-Radecki) [reply eth/contracts/StableSwap/StableSwapTwoPool.sol:413]: I'm good now thanks

**Review by Marcin-Radecki — APPROVED** [review]

**Pull request merged**